### PR TITLE
privateEndpoint_ARM_Template

### PR DESCRIPTION
**Pull request by v-gsapaharam**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by v-gsapaharam** [comment]
Hi Mahender, could you please review.

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateendpoint/azuredeploy.parameters.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```
Comment (bmoore-msft) [inline quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateendpoint/azuredeploy.parameters.json:4]: remove the param values from here and use defaultValues in the template

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
What are all the PNG files for?  They don't seem to be used anywhere...

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateendpoint/azuredeploy.parameters.json (hunk 1 above)
Comment (v-gsapaharam) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateendpoint/azuredeploy.parameters.json:4]: Thanks, @bmoore-msft for the quick review.

**Comment by v-gsapaharam** [comment]
> What are all the PNG files for? They don't seem to be used anywhere...

This has been taken care.

File: quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateendpoint/azuredeploy.parameters.json (hunk 1 above)
Comment (bmoore-msft) [reply quickstarts/microsoft.databricks/databricks-all-in-one-template-for-vnet-injection-privateendpoint/azuredeploy.parameters.json:4]: This still needs to be done

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
Also check the pipeline results for other changes that need to be made

**Comment by bmoore-msft** [comment]
@v-gsapaharam - still need to get the pipeline errors cleaned up...

**Comment by v-gsapaharam** [comment]
> @v-gsapaharam - still need to get the pipeline errors cleaned up...

Need some help resolving the errors about svg files.

**Comment by v-gsapaharam** [comment]
> @v-gsapaharam - still need to get the pipeline errors cleaned up...

Need some pointers, how we can make svg files reference in "Readme.md" specific to our template

**Comment by v-gsapaharam** [comment]
> Also check the pipeline results for other changes that need to be made

@bmoore-msft - All pipeline/build issues resolved, could you please review once.

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can get a passing ci

**Pull request closed without merging**